README.md

```diff
@@ -82,35 +81,70 @@ The following tables show the advised use cases for each algorithm. The column *
+| Likelihood Ratios     | ✔       | ✔     |  ✔          |      |  ✔                   |       |  ✔            |
+| Prophet               |        |      |  ✔          |      |                     |       |              |
+| Spectral Residual     |        |      |  ✔          |      |                     |  ✔     |  ✔            |
+| Seq2Seq               |        |      |  ✔          |      |                     |       |  ✔            |
 
 ### Adversarial Detection
 
 | Detector           | Tabular | Image | Time Series | Text  | Categorical Features | Online | Feature Level |
 | :---               |  :---:  | :---: |   :---:     | :---: |   :---:              | :---:  | :---:         |
-| Adversarial AE     | ✔       | ✔     |  ✘          |  ✘    |  ✘                   |  ✘     |  ✘            |
-| Model distillation | ✔       | ✔     |  ✔          |  ✔    |  ✔                   |  ✘     |  ✘            |
+| Adversarial AE     | ✔       | ✔     |            |      |                     |       |              |
+| Model distillation | ✔       | ✔     |  ✔          |  ✔    |  ✔                   |       |              |
 
 
 ### Drift Detection
 
 | Detector                 | Tabular | Image | Time Series | Text  | Categorical Features | Online | Feature Level |
 | :---                     |  :---:  | :---: |   :---:     | :---: |   :---:              | :---:  | :---:         |
-| Kolmogorov-Smirnov       | ✔       | ✔     |  ✘          |  ✔    |  ✔                   |  ✘     |  ✔            |
-| Maximum Mean Discrepancy | ✔       | ✔     |  ✘          |  ✔    |  ✔                   |  ✘     |  ✘            |
-| Chi-Squared              | ✔       | ✘     |  ✘          |  ✘    |  ✔                   |  ✘     |  ✔            |
-| Mixed-type tabular data  | ✔       | ✘     |  ✘          |  ✘    |  ✔                   |  ✘     |  ✔            |
-| Classifier               | ✔       | ✔     |  ✔          |  ✔    |  ✔                   |  ✘     |  ✘            |
+| Kolmogorov-Smirnov       | ✔       | ✔     |            |  ✔    |  ✔                   |       |  ✔            |
+| Maximum Mean Discrepancy | ✔       | ✔     |            |  ✔    |  ✔                   |       |              |
+| Chi-Squared              | ✔       |      |            |      |  ✔                   |       |  ✔            |
+| Mixed-type tabular data  | ✔       |      |            |      |  ✔                   |       |  ✔            |
+| Classifier               | ✔       | ✔     |  ✔          |  ✔    |  ✔                   |       |              |
+
+#### TensorFlow and PyTorch support
+
+The drift detectors support TensorFlow and PyTorch backends. Example:
+
+```python
+from alibi_detect.cd import MMDDrift
+
+cd = MMDDrift(x_ref, backend='tensorflow', p_val=.05)
+preds = cd.predict(x)
+```
+
+The same detector in PyTorch:
+
+```python
+cd = MMDDrift(x_ref, backend='pytorch', p_val=.05)
+preds = cd.predict(x)
+```
```

Comment:
Add a sentence saying that `pytorch` should be installed separately, same for examples when talking about `pytorch`.